src/ibex_bluesky_core/callbacks/_utils.py

```diff
@@ -3,8 +3,7 @@
 from platform import node
 
 INSTRUMENT = node()
-
-OUTPUT_DIR_ENV_VAR = "IBEX_BLUESKY_OUTPUT_DIR"
+OUTPUT_DIR_ENV_VAR = "IBEX_BLUESKY_CORE_OUTPUT"
```

Comment:
as this is done at import-level i think it's not possible to change the value of this env var and get it picked up dynamically ie. just before running a plan